_posts/2022-04-07-launching-open-source-operator-for-openSearch.markdown

```diff
@@ -0,0 +1,71 @@
+
+The [OpenSearch Operator](https://github.com/Opster/opensearch-k8s-operator) is fully open-source kubernetes operator, licensed as Apache 2.0, and is used for automating the deployment, provisioning, management, and orchestration of OpenSearch clusters and OpenSearch dashboards.
+
+The Operator development is being led by [Opster](https://opster.com/) with partners including [SUSE Rancher](https://www.suse.com/), [Maibornwolff](https://www.maibornwolff.de/en), [AWS](https://aws.amazon.com/), [Logz.io](https://logz.io/) and more.
+
+### Operator Capabilities
+
+The Operator enables high-level API use for easily running advanced OpenSearch operations on Kubernetes.
+
+With this beta release, the Operator allows for management of multiple OpenSearch clusters and OpenSearch Dashboards. Using Operator makes scaling up and down, version upgrades, rolling restarts, adjustment of memory and disk resources on the nodes, securing deployments and managing certificates simplified and streamlined.
+
+In the future releases, it would also allow for advanced shard allocation strategies, monitoring with Prometheus and Grafana, control shard balancing and allocation (For example based on AZ/Rack awareness, Hot/Warm) and auto-scaling based on usage load and resources.
+
+### Getting Started
+The Operator is available at https://github.com/Opster/opensearch-k8s-operator.
+
+#### Step 1: Installation
+In order to install the Operator you can use 2 methods:
+    1. Helm chart and the other is to build it locally.
+    2. Local installation
+
+###### Helm chart installation
+- Download the helm gz file from https://github.com/Opster/opensearch-k8s-operator/releases/download/v0.9/opensearchOperator-chart.tar.gz
+- Follow these instructions: https://github.com/Opster/opensearch-k8s-operator#installing-the-operator-on-your-k8s-cluster-with-helm
+
+###### Local installation
+- Clone the repo
+- Run `make build manifests` to build the controller binary and the manifests
+- Start a kubernetes cluster (e.g. with k3d or minikube) and make sure your `~/.kube/config` points to it
+- Run `make install` to create the CRD in the kubernetes cluster
+
+There is no special recommandation on which method to use, both of them will install the OpenSearch operator deployment and CRD on your kubernetes cluster. If you would like to explore the source code and play with it you can use method #2.
+
+#### Step 2: Deploying a new OpenSearch cluster
+
+Go to `opensearch-operator` and use `opensearch-cluster.yaml` as a starting point to define your cluster - note that the `clusterName` is also the namespace that the new cluster will reside in. Then run:
+
+```bash
+kubectl apply -f opensearch-cluster.yaml
+```
+The current installation deploys with the default demo certificate provided by OpenSearch which are good for demo purposes but not safe for production use. We recommend replacing the demo certificates with a trusted CA provided certificates for any production use.
```

Comment:
Please add a comma after "OpenSearch". Change "which are" to "which is". I recommend replacing "good" with a word like "ideal" or "acceptable", depending on your meaning. Second line: do we mean "with trusted CA-provided certificates"? Unless it is being treated as commonly understood by the reader, please define "CA" as "certificate authority (CA)". With both changes implemented, this would be "with trusted certificate authority (CA)-provided certificates".